polaris-react/src/components/InContextLearning/InContextLearning.tsx

```diff
@@ -109,42 +99,46 @@ export function InContextLearning({children, steps, onDismiss}: Props) {
           return (
             <>
               <div
+                className={styles.InContextLearning}
                 style={{
-                  padding: '1em',
-                  backgroundColor: '#fff',
-                  filter:
-                    'drop-shadow(0px 0px 3px rgba(0, 0, 0, 0.1)) drop-shadow(0px 4px 20px rgba(0, 0, 0, 0.15))',
                   ...verticalMargin,
                 }}
               >
                 <KeypressListener keyCode={Key.Escape} handler={onDismiss} />
-                {dismissButton}
-                {steps[currentStep].content}
-                <Stack
-                  alignment="center"
-                  wrap={false}
-                  distribution="fillEvenly"
-                >
-                  <Stack.Item>
+                <Header title={title} onDismiss={onDismiss} />
+                <div className={styles.TutorialContent}>
+                  {steps[currentStep].content}
+                </div>
+                <div className={styles.TutorialFooter}>
+                  <Stack
```

Comment:
Personally, I hate the stack component and find it hard to style

Reply:
I don't mind `stack`, and we can do without any custom styling here to match the [mocks](https://www.figma.com/file/gAysnv8y94Dj3V4ZKoykdW/Prep-work?node-id=140%3A2898):

```jsx
<Stack alignment="center" wrap={false} distribution="fillEvenly">
  <Stack.Item>
    {hasMultipleSteps && (
      <TextStyle variation="subdued">
        {currentStep + 1} of {totalSteps}
      </TextStyle>
    )}
  </Stack.Item>
  <Stack.Item>
    <Stack
      alignment="center"
      wrap={false}
      spacing="tight"
      distribution="trailing"
    >
      {showBack && <Button onClick={handleBack}>Back</Button>}
      {showNext && (
        <Button primary onClick={handleNext}>
          Next
        </Button>
      )}
      {showClose && (
        <Button primary onClick={handleClose}>
          Got it
        </Button>
      )}
    </Stack>
  </Stack.Item>
</Stack>;
```